internal/changelog/changelog.go

```diff
@@ -145,12 +182,20 @@ func (c *Changelog) GenerateChangelog(templateConfig shared.ChangelogTemplateCon
 	}
 
 	commitsContent := commitsContent{
-		Commits:         commitsPerScope,
-		BreakingChanges: commitsBreakingChange,
-		Backtick:        "`",
-		Order:           order,
-		HasURL:          templateConfig.CommitURL != "",
-		URL:             templateConfig.CommitURL,
+		Commits:          commitsPerScope,
+		BreakingChanges:  commitsBreakingChange,
+		Backtick:         "`",
+		Order:            order,
+		ShowBodyAsHeader: c.config.Changelog.ShowBodyAsHeader,
+		HasURL:           templateConfig.CommitURL != "",
+		URL:              templateConfig.CommitURL,
+	}
+
+	authorsNames := make([]string, len(authors))
```

Comment:
also specify the cap. of the array. If not specified the backing underlying array will have a capacity of 2x the given length I think 🤪

Reply:
Nope im wrong, when omitted will be the the same as the given length.